scripts/readme.md

```diff
@@ -56,69 +56,3 @@ Arguments can be
 You can also arbitrary arguments in the command line, and the arguments will save in a dictionary and merge with dataset/model/pipeline's existing cfg.
 For example, `--foo abc` will add `{"foo": "abc"}`to the cfg dict.
 
```

Comment:
You just removed all this information and left this 2 lines to explain it? C'mon... 
Also let's make readme.md --> README.md

Reply:
I discussed this with benjamin, and we think this doesn't add much value. the description of using `semseg.py` may be  sufficient. What do you think?